sklearn/metrics/pairwise.py

```diff
@@ -720,6 +720,72 @@ def haversine_distances(X, Y=None):
     return DistanceMetric.get_metric('haversine').pairwise(X, Y)
 
 
+def pairwise_wasserstein_distances(X, Y, X_d, Y_d):
```

Comment:
It seems inappropriate to have a different signature from our other distance functions. Is it appropriate instead to treat X and Y as being the concatenation of means and their variances?

Reply:
sure, let me think about how to make this more consistent with other function signatures. 
thanks, will make a modified PR soon.